libs/utils/test.py

```diff
@@ -42,20 +42,31 @@ def _init(cls, conf, *args, **kwargs):
             cls.logger.setLevel(kwargs['loglevel'])
             kwargs.pop('loglevel')
 
-        cls.conf = conf
-
         cls._runExperiments()
 
     @classmethod
+    def _getTestConf(cls):
+        if not hasattr(cls, "test_conf"):
+            raise NotImplementedError("Please add `test_conf` attribute")
+        return cls.test_conf
+
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
```

Comment:
It is not clear why we expect test_env or what a derived class can use it for.

We should probably add a bit of documentation at least to this two methods.